mon-pix/app/components/challenge-item-qrocm.js

```diff
@@ -27,12 +29,7 @@ class ChallengeItemQrocm extends ChallengeItemGeneric {
   // This is not "the Ember way", however it makes code easier to read,
   // and moreover, is a much more robust solution when you need to test it properly.
   _getRawAnswerValue() {
-    const result = {};
-    // XXX : forEach on NodeList returned by document.querySelectorAll is not supported by IE
-    _.forEach(document.querySelectorAll('.challenge-proposals input'), (element) => {
-      result[element.getAttribute('name')] = element.value;
-    });
-    return result;
+    return this.answerValues;
```

Comment:
Du coup, la fonction `_getRawAnswerValue` peut être remplacée par `this.answerValues` directement.